test/unittests/state_transition.cpp

```diff
@@ -92,12 +100,6 @@ void state_transition::TearDown()
         EXPECT_EQ(trace_capture->get_capture(), expect.trace);
     }
 
-    EXPECT_EQ(receipt.status, expect.status);
-    if (expect.gas_used.has_value())
-    {
-        EXPECT_EQ(receipt.gas_used, *expect.gas_used);
-    }
-
```

Comment:
Can you explain why it's removed?

Reply:
not removed, just moved to the `else` clause above